web-react/src/App.tsx

```diff
@@ -0,0 +1,53 @@
+import { useState } from "react";
+import Register from "@/components/tabs/Register";
+import Faucet from "@/components/tabs/Faucet";
+import Transfer from "@/components/tabs/Transfer";
+import Approve from "@/components/tabs/Approve";
+import Swap from "@/components/tabs/Swap";
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+
+enum TabOption {
+  Register = "Register",
+  Faucet = "Faucet",
+  Transfer = "Transfer",
+  Approve = "Approve",
+  Swap = "Swap",
+}
+
+const TabComponents: Record<TabOption, React.FC> = {
+  [TabOption.Register]: () => <Register />,
+  [TabOption.Faucet]: () => <Faucet />,
+  [TabOption.Transfer]: () => <Transfer />,
+  [TabOption.Approve]: () => <Approve />,
+  [TabOption.Swap]: () => <Swap />,
+};
+
+const queryClient = new QueryClient();
+
+function App() {
+  const [activeTab, setActiveTab] = useState<TabOption>(TabOption.Approve);
```

Comment:
I prefer default tab to be the Faucet one :pray: 

```suggestion
  const [activeTab, setActiveTab] = useState<TabOption>(TabOption.Faucet);
```